controllers/drplacementcontrol_controller.go

```diff
@@ -1759,3 +1761,30 @@ func addOrUpdateCondition(conditions *[]metav1.Condition, conditionType string,
 
 	return false
 }
+
+// Initial creation of the DRPC status condition. This will also preserve the ordering of conditions in the array
+func ensureDRPCConditionsInited(conditions *[]metav1.Condition, observedGeneration int64, message string) {
+	const DRPCTotalConditions = 2
+	if len(*conditions) == DRPCTotalConditions {
```

Comment:
This constant is not helping, it does not provide more info and it is not used elsewhere.

Reply:
Right. It is not used anywhere, but it is helping a little bit in two ways:
1. It is readable. IOW, now you know what 2 means. And the const tells you that it is really for readability only and the value of 2 is a constant expectation. Meaning, the condition array can't have more than 2 or less than 2.
2. Staisfies the linter.

Reply:
We absolutely can. Every time you stare at a block of code for a long time, you end up thinking of a better way to rewrite it. This is just the nature of software engineering. At this moment, I will leave it as is for this PR.